common/src/main/java/com/dfsek/terra/biome/UserDefinedBiome.java

```diff
@@ -1,55 +1,52 @@
 package com.dfsek.terra.biome;
 
+import com.dfsek.terra.api.math.ProbabilityCollection;
+import com.dfsek.terra.api.platform.world.Biome;
 import com.dfsek.terra.api.platform.world.World;
-import com.dfsek.terra.api.world.biome.Biome;
 import com.dfsek.terra.api.world.biome.Generator;
-import com.dfsek.terra.config.base.ConfigPack;
 import com.dfsek.terra.config.builder.GeneratorBuilder;
 import com.dfsek.terra.config.templates.BiomeTemplate;
 
+import java.util.HashSet;
+import java.util.Set;
+
 /**
  * Class representing a config-defined biome
  */
-public class UserDefinedBiome implements Biome {
+public class UserDefinedBiome implements TerraBiome {
     private final GeneratorBuilder gen;
-    private final com.dfsek.terra.api.platform.world.Biome vanilla;
+    private final ProbabilityCollection<Biome> vanilla;
     private final String id;
     private final BiomeTemplate config;
-    private final ConfigPack pack;
-    private UserDefinedBiome erode;
+    private final int color;
+    private final Set<String> tags;
 
 
-    public UserDefinedBiome(com.dfsek.terra.api.platform.world.Biome vanilla, GeneratorBuilder gen, BiomeTemplate config, ConfigPack pack) {
+    public UserDefinedBiome(ProbabilityCollection<Biome> vanilla, GeneratorBuilder gen, BiomeTemplate config) {
         this.vanilla = vanilla;
         this.gen = gen;
         this.id = config.getID();
         this.config = config;
-        this.pack = pack;
+        this.color = config.getColor();
+        this.tags = config.getTags() == null ? new HashSet<>() : config.getTags();
+        tags.add("BIOME:" + id);
     }
 
     /**
      * Gets the Vanilla biome to represent the custom biome.
      *
-     * @return Biome - The Vanilla biome.
+     * @return TerraBiome - The Vanilla biome.
      */
     @Override
-    public com.dfsek.terra.api.platform.world.Biome getVanillaBiome() {
+    public ProbabilityCollection<Biome> getVanillaBiomes() {
         return vanilla;
     }
```

Comment:
Documentation says a TerraBiome is returned. In reality the method returns a `ProbabilityCollection<Biome>`. The code or the documentation is in error.